test/common/integration/rtds_integration_test.cc

```diff
@@ -62,166 +26,41 @@ envoy::config::bootstrap::v3::LayeredRuntime layeredRuntimeConfig(const std::str
-    sotw_or_delta_ = sotwOrDelta();
-    scheme_ = "https";
-
-    if (sotw_or_delta_ == Grpc::SotwOrDelta::UnifiedSotw ||
-        sotw_or_delta_ == Grpc::SotwOrDelta::UnifiedDelta) {
-      config_helper_.addRuntimeOverride("envoy.reloadable_features.unified_mux", "true");
-    }
-
-    // Set up the cluster config.
-    //
-    // For now, we clear the default cluster configs and add just two clusters:
-    //   - a cluster named "base_h2" because that's what the api_listener is configured to talk to
-    //   - an RTDS cluster, for sending and receiving RTDS config
-    //
-    // The reason we must clear the default cluster configs is because ConfigHelper::setPorts
-    // requires that the number of fake upstream ports equal the number of clusters in the config
-    // that have dynamic port configuration (i.e. port is 0). In other words, either all fake
-    // upstreams must be configured with a dynamic port or none of them (can't mix and match).
-    //
-    // TODO(abeyad): fix the ConfigHelper::setPorts logic to enable a subset of clusters to have
-    // dynamic port configuration.
-
-    config_helper_.addConfigModifier([this](envoy::config::bootstrap::v3::Bootstrap& bootstrap) {
-      const std::string loopback = loopbackAddr();
-      bootstrap.mutable_static_resources()->clear_clusters();
-      bootstrap.mutable_static_resources()->add_clusters()->MergeFrom(
-          createSingleEndpointClusterConfig("base_h2", loopback));
-      bootstrap.mutable_static_resources()->add_clusters()->MergeFrom(
-          createSingleEndpointClusterConfig("rtds_cluster", loopback));
-    });
-
-    // xDS upstream is created separately in the test infra, and there's only one non-xDS cluster.
-    setUpstreamCount(1);
-
-    // Add the Admin config.
-    config_helper_.addConfigModifier([this](envoy::config::bootstrap::v3::Bootstrap& bootstrap) {
-      bootstrap.mutable_admin()->MergeFrom(adminConfig(loopbackAddr()));
-    });
-    admin_filename_ = TestEnvironment::temporaryPath("admin_address.txt");
-    setAdminAddressPathForTests(admin_filename_);
-  }
-
-  void SetUp() override {
-    // TODO(abeyad): Add paramaterized tests for HTTP1, HTTP2, and HTTP3.
-    setUpstreamProtocol(Http::CodecType::HTTP2);
-  }
-
-  void TearDown() override {
-    cleanup();
-    BaseClientIntegrationTest::TearDown();
-  }
-
-  void createEnvoy() override {
-    BaseClientIntegrationTest::createEnvoy();
-    std::string admin_str = TestEnvironment::readFileToStringForTest(admin_filename_);
-    auto addr = Network::Utility::parseInternetAddressAndPort(admin_str);
-    registerPort("admin", addr->ip()->port());
-  }
-
-  void initialize() override {
-    BaseClientIntegrationTest::initialize();
-    // Register admin port.
-    // registerTestServerPorts({});
-
-    acceptXdsConnection();
-  }
-
-  void addRuntimeRtdsConfig() {
-    // Add the layered runtime config, which includes the RTDS layer.
+  RtdsIntegrationTest() : XdsIntegrationTest() {
```

Comment:
C++ does the right thing since it's a zero arg constructor so it automatically calls the parent class' zero-arg constructor.  I removed the explicit call.